src/coreclr/pal/inc/pal.h

```diff
@@ -2847,19 +2847,6 @@ VirtualQuery(
 #define LMEM_ZEROINIT       0x0040
```

Comment:
These flags are for use with LocalAlloc. They can be deleted too.

Reply:
Deleted. Also noticed that there is only one usage of PAL_dladdr wrapper in module.cpp, where we already using dladdr directly, so I have deleted that wrapper as well.